explainers/update-rendering.md

```diff
@@ -16,38 +22,62 @@ words, if the website intends to show an element whose contents are currently
 
-The function returns a promise which resolves when the contents of the element
-are ready to be displayed to screen.  Note that repeated calls to
-updateRendering() returns new promises, with the ultimate priority of the
-request being the highest requested priority.
+* `auto` is the default priority, which should be equivalent to `userBlocking`
+  for the purposes of User Agent prioritization.
+
+Note that the attribute values follow [the postTask API](https://wicg.github.io/scheduling-apis/#sec-task-priorities)
+example.
+
+
+### Interaction with `content-visibility`
+Since the User Agent typically keeps rendering state of subtrees up-to-date,
+this feature would be a no-op in a majority of cases. For instance, having
+`renderPriority` on a visible, on screen, element would not have to do any
+work since the rendering state of such an element is already kept up to date.
+
+We're proposing this feature as an enhancement for the `content-visibility` CSS
+property. For example, when `content-visibity: hidden` property is applied to
+the element, it ensures that the element's subtree is not visible and that its
+rendering state is not updated to save CPU time. Adding `renderPriority` on
+such an element, however, would cause the User Agent to continually process its
+rendering with a given priority.
+
+### Notes and Clarifications
+* If the User Agent does not optimize rendering of elements, by skipping work,
+  then the attribute has no effect.
 
-### Notes
-If the User Agent does not optimize rendering of elements, then the
-function can return a resolved promise for every call. Note that a successful
-resolution of the promise does not provide strong guarantees of performance:
-bringing the element on screen may partially or fully invalidate the work that
-has been prepared.  The User Agent should prepare the element and its contents
-as much as it deems is necessary for presentation
+* Setting the attribute on an element whose rendering state is not updated due
+  to `display: none`, ancestor style that prevents update (e.g. ancestor
+  `content-visibility: hidden`), or similar styles has no effect. In other
+  words, this attribute would not force rendering state to be updated on
+  elements whose rendering work is required to be skipped. ([more
+  details](https://github.com/WICG/display-locking/issues/199)). Note that this
+  does _not_ include elements that themselves have `content-visibility: hidden`
+  style, since their rendering is updated and the attribute would cause the
+  subtree to be udpated as well.
 
+* Setting the attribute on an element that contains descendants with
+  `content-visibility: hidden`, `display: none`, or similar styles would not
+  cause such descendants to be updated. This is a consequence of the fact that
+  a fully updated parent element has all its rendering work completed without
+  updating such descendants. ([more
+  details](https://github.com/WICG/display-locking/issues/196))
```

Comment:
```suggestion
  details](https://github.com/WICG/display-locking/issues/196)). If such an update is desired, a `renderPriority` attribute should be set on such elements.
```

Should we add a sentence saying something like "a recursive `renderPriority` setting may be considered in the future?